### PR TITLE
fix the bug of returning array of nil

### PR DESCRIPTION
**Pull request by mengwan**

### CODE REVIEW

**Comment by loganb** [comment]
Awesome! Thanks for the PR. Would you mind adding some specs to the test suite that validate this behavior?

**Comment by loganb** [comment]
Looks good! Will fixup the conflicts and merge soon.

**Comment by philipmw** [comment]
Would you please rebase this to a single commit?

File: lib/dynamoid/finders.rb [hunk 1]
```diff
@@ -16,15 +16,18 @@ module ClassMethods
       #
       # @since 0.2.0
       def find(*ids)
+
```
Comment (philipmw) [inline lib/dynamoid/finders.rb]: spurious empty line

**Comment by philipmw** [comment]
Thanks for your contribution!

**Pull request merged**